include/deal.II/base/data_out_base.h

```diff
@@ -2708,9 +2708,9 @@ class DataOutInterface
    * parallelize visualization. In that case, you need a
    * <code>.pvtu</code> file that describes which VTU files (written, for
    * example, through the DataOutInterface::write_vtu() function) form a group.
-   * The current function can generate such a master record.
+   * The current function can generate such a centralized record.
    *
-   * The master record file generated by this function
+   * The main record file generated by this function
```

Comment:
You seem to replace master seemingly randomly by "parent", "centralized", and "main" here (where it relates to the pvtu files). I find that confusing.

Reply:
I changed this to "central record", since that now coincides with @bangerth's suggested change below.